packages/synchro-charts/src/components/sc-dial/sc-dial-base/util.ts

```diff
@@ -0,0 +1,81 @@
+export enum StatusProgress {
+  BLUE = '#2E72B5',
+  NORMAL = '#3F7E23',
+  WARNING = '#F29D38',
+  CRITICAL = '#C03F25',
+  GRAY = '#D9D9D9',
+  PRIMARYTEXT = '#16191f',
+  SECONDARYTEXT = '#687078',
+}
+
+export const enum SizeFont {
+  Huge = 'Huge',
+  XL = 'XL',
+  L = 'L',
+  M = 'M',
+  S = 'S',
+  XS = 'XS',
+}
+
+export interface TextSizeConfig {
+  value: number;
+  unit: number;
+  label: number;
+  alarm: number;
+}
+
+export const sizeContent = {
+  Huge: {
+    value: 96,
+    unit: 48,
+    label: 48,
+    alarm: 48,
+  },
+  XL: {
+    value: 60,
+    unit: 32,
+    label: 32,
+    alarm: 32,
+  },
+  L: {
+    value: 48,
+    unit: 24,
+    label: 20,
+    alarm: 24,
+  },
+  M: {
+    value: 32,
+    unit: 20,
+    label: 20,
+    alarm: 20,
+  },
+  S: {
+    value: 24,
+    unit: 16,
+    label: 16,
+    alarm: 16,
+  },
+  XS: {
+    value: 20,
+    unit: 14,
+    label: 16,
+    alarm: 14,
+  },
+};
+
+export const LengthToSize = {
```

Comment:
What exactly does this mean? Add comment if variable name can't fully convey. I'm guessing this means 'number of characters in a string maps to a particular font size', but only because I've read through the code

Reply:
Done --- Add comment